### ISSUE TITLE
Timed out waiting for systemd to enter running state.

### ISSUE THREAD

**Issue by TylerJaacks** [arkane-systems/genie#229]
**Windows version (build number):**
Microsoft Windows
Version 21H2 (OS Build 22000.527)

**Linux distribution:**
Ubuntu from the Windows Store updated to 21.10

**Genie version:**
1.44

**Describe the bug**
```
Waiting for systemd....!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Timed out waiting for systemd to enter running state.
This may indicate a systemd configuration error.
Attempting to continue.
Failed units will now be displayed (systemctl list-units --failed):
Failed to list units: Transport endpoint is not connected
genie: running command failed; nsenter for systemctl list-units --failed returned 1.

**Confirm that you are running inside the bottle:**
The output of `genie -b`.
```

**To Reproduce**
genie -s

**Expected behavior**
It works

 - [ x ] Yes.

**Comment by cerebrate** [comment]
Rather than debugging a now-obsolete version, mind giving it a try with the now-published 2.2?

**Comment by cerebrate** [comment]
Closed as stale.
